lib/model/internal_link.dart

```diff
@@ -292,3 +303,28 @@ List<int>? _parseDmOperand(String operand) {
     return null;
   }
 }
+
+/// An operand value of "is" operator.
+///
+/// See also:
+///   - https://zulip.com/api/construct-narrow
+///   - https://zulip.com/help/search-for-messages#search-your-important-messages
+///   - https://zulip.com/help/search-for-messages#search-by-message-status
```

Comment:
Cool, thanks for the thorough collection of links — this made it easy for me to verify the list of values.